src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -3371,8 +3373,36 @@ class ArnoldGlobals
 			const IECore::Msg::Level level = \
 				( logmask == AI_LOG_DEBUG ) ? IECore::Msg::Level::Debug : g_ieMsgLevels[ min( severity, 3 ) ];
 
-			g_currentMessageHandler->handle( level, "Arnold", std::string( tabs, ' ' ) + msg_string );
+			std::stringstream msg;
+
+			const int flags = AiMsgGetConsoleFlags();
+			if( flags & AI_LOG_TIMESTAMP )
+			{
+				const boost::posix_time::time_duration elapsed = boost::posix_time::millisec( AiMsgUtilGetElapsedTime() );
+				msg << std::setfill( '0' );
+				msg << std::setw( 2 ) << elapsed.hours() << ":";
+				msg << std::setw( 2 ) << elapsed.minutes() << ":";
+				msg << std::setw( 2 ) << elapsed.seconds() << " ";
+			}
+			if( flags & AI_LOG_MEMORY )
+			{
+				const size_t mb = AiMsgUtilGetUsedMemory() / 1024 / 1024;
+				if( mb < 1024 )
+				{
+					msg << mb << "MB  ";
+				}
+				else
+				{
+					msg.setf( std::ios::fixed, std:: ios::floatfield );
```

Comment:
`std:: ios` -> `std::ios`

Reply:
I just moved the space from here into the earlier comment to fix the padding :-) save waste (fixup)[https://github.com/GafferHQ/gaffer/pull/3940/commits/73cd15cc570bc5732d86a8393425437e046ae984].